source/concepts/architecture.rst

```diff
@@ -138,6 +140,18 @@ For more information, see `Installing PMM Server <https://www.percona.com/doc/pe
 All tools can be accessed from the |pmm-server| web interface (landing page).
 For more information, see :ref:`using`.
 
+.. _pmm-platform:
+
+`PMM Platform <architecture.html#pmm-platform>`_
+================================================================================
+
+|pmm-platform| provides the following value-added services to |pmm|.
+
+|stt|
+-----------------------------------------------
+
+|stt| checks registered database instances for a range of common security issues. This service requires the Telemetry setting to be on (see :ref:`server-admin-gui-telemetry`).
```

Comment:
It also requires STT setting to be on – STT is a separate toggle from Telemetry that requires explicit opt-in. It is off by default, unlike Telemetry which is opt-out.
STT toggle is being implemented, but it will be a part of PMM 2.6 release – we can't release it without that toggle.